walkthroughs/week-2-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java

```diff
@@ -26,7 +26,11 @@
 @WebServlet("/server-stats")
 public final class ServerStatsServlet extends HttpServlet {
 
-  private final Date startTime = new Date();
+    /**
+     *
+     */
+    private static final long serialVersionUID = 1L;
```

Comment:
Check the indentation for both these variables and remove the /***/ above if you're not adding any comments.

Reply:
<img width="1320" alt="Screen Shot 2021-03-07 at 2 40 44 AM" src="https://user-images.githubusercontent.com/79274071/110233001-3b2cb000-7eef-11eb-9d7e-151a7b891d1c.png">

a little gross haha but it works so yay. Also, thank you for the correction I got rid of the comments.

Reply:
This looks good, thanks! One thing, notice how your image is not loading. Make sure you include the image resource file and you use the correct path.

Reply:
okay thank you